### PR TITLE
Bug/#443 partner application filter views

### PR DESCRIPTION
**Pull request by kenzo23312**

### CODE REVIEW

File: frontend/src/components/eois/filters/eoiHelper.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-
-const resetChanges = (pathName, query) => {
-  const { page } = query;
-  const { page_size } = query;
 
+const resetChanges = (pathName) => {
   history.push({
     pathname: pathName,
-    query: { page, page_size },
+    query: { page: 1, page_size: 10 },
```
Comment (marcindo) [inline frontend/src/components/eois/filters/eoiHelper.js]: why do you want to reset pagination to default on filter reset?

File: frontend/src/components/eois/modals/newCfei/newCfeiModal.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
+import R from 'ramda';
```
Comment (marcindo) [inline frontend/src/components/eois/modals/newCfei/newCfeiModal.js]: why all the changes here if none of it is used?

File: frontend/src/components/partners/partnerProfileCountriesCell.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { TableCell } from 'material-ui/Table';
+import withCountryName from '../common/hoc/withCountryName';
+import EoiCountryCell from '../eois/cells/eoiCountryCell';
+
+const PartnerProfileCountriesCell = (props) => {
+  const { countries } = props;
+
+  return (
+    <TableCell>
+      {countries && countries.map(item => (<div><EoiCountryCell code={item} />{', '}</div>))}
```
Comment (marcindo) [inline frontend/src/components/partners/partnerProfileCountriesCell.js]: isn't it better to just do the map inside cells template? I'm doing same thing for eoiTable, so it's either one common component with map of countries, or map directly everywhere

File: frontend/src/components/eois/filters/eoiHelper.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/components/eois/filters/eoiHelper.js]: done

File: frontend/src/components/eois/modals/newCfei/newCfeiModal.js (hunk 2 above)
Comment (kenzo23312) [reply frontend/src/components/eois/modals/newCfei/newCfeiModal.js]: done

File: frontend/src/components/partners/partnerProfileCountriesCell.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TableCell } from 'material-ui/Table';
+import withCountryName from '../common/hoc/withCountryName';
+import EoiCountryCell from '../eois/cells/eoiCountryCell';
+
+const PartnerProfileCountriesCell = (props) => {
```
Comment (marcindo) [inline frontend/src/components/partners/partnerProfileCountriesCell.js]: Please change the name to more generic as it is not related to partner profile

**Pull request merged**